tutor/src/components/annotations/annotation.jsx

```diff
@@ -151,14 +152,22 @@ export default class AnnotationWidget extends React.Component {
       }
       this.ux.statusMessage.hide();
     };
-    return imagesComplete({
-      body: this.props.windowImpl.document.querySelector('.book-content'),
-    }).then(initialize).catch(initialize);
-
+    const win = this.props.windowImpl;
+    const runImagesComplete = () => {
+      imagesComplete({
+        body: win.document.querySelector('.book-content'),
+      }).then(initialize).catch(initialize);
+    };
+    if (win.MathJax) {
+      win.MathJax.Hub.Register.MessageHook('End Process', runImagesComplete);
```

Comment:
Hm, that's a great point.  It could be possible that mathjax hasn't been started, which would mean that "End Process" would never be called.

However since annotations are only used on html content, mathjax will always have been ran.  Still it's best not to rely on that, so I'll change the iff to be:
`if (win.MathJax && win.MathJax.Hub.queue.running) {`